app/services/catalog/tenant_settings.rb

```diff
@@ -0,0 +1,28 @@
+module Catalog
+  class TenantSettings
+    attr_reader :response
+
+    def initialize(tenant)
+      @tenant = tenant
+    end
+
+    def process
+      @response = {
+        :current => @tenant.settings,
+        :schema  => JSON.parse(schema)
+      }
+
+      # TODO: these things:
+      # 1. set schema default_workflow enum to whatever the possible approved workflows are
+      # 2. fetch default approval and include in list as well
+
```

Comment:
Do any of these need to be done before the PR is considered 'done'?

Reply:
Ahhh yep my bad, I'll remove these. They're handled by the schema stuff down here https://github.com/ManageIQ/catalog-api/pull/328/files#r302781806